src/index.js

```diff
@@ -200,8 +349,16 @@ const renderLevel = (root, levelNumber) => {
         renderBoard();
 
         if (soko.solved()) {
-          setTimeout(() => { renderLevelComplete(root); }, 100);
-          console.log("solved!");
+          const score = localStorage.getItem(levelNumber+1);
+          if (score) {
+            if (score > soko.movesSize()){
+              updateBestScore(levelNumber+1, soko.movesSize());
+            }
+          } 
+          else if (!score) {
+            updateBestScore(levelNumber+1, soko.movesSize());
```

Comment:
```suggestion
            updateBestScore(levelNumber + 1, soko.movesSize());
```

Missing some spaces around operators throughout.